scripts/GiffyLube.js

```diff
@@ -8,29 +12,49 @@ import { NavBarHTML } from "./nav/NavBar.js"
 // import getView function from ./data/TransientState.js
 
 /////////////////////////////////////////////////////////////////////////////////////////////////////////////
-
-/* 
-    Define and export GiffyLube function that generates all the html for main feed page
-    Don't forget to await functions with embedded fetch calls
-
-    Create variable for htmlString (or another name) = NavBar //////////////
-
-    check these conditions of the transientState to determine which page to generate (only one or none should be true)
-        1. If transientState.view===viewMessages, then invoke function that generates all the HTML for the inbox page and add it to the htmlString
-        2. If transientState.view===createMessage, then invoke function that generates all the HTML for the create message page and add it to the htmlString
-        3. If transientState.view===postGif, then invoke function that generates all the HTML for the new post page and add it to the htmlString
-        4. If transientState.view===defaultView, generate the HTML for the main feed page and add it to the htmlString
-*/
+//    Define and export GiffyLube function that generates all the html for main feed page
 export const GiffyLube = async () => {
     const navBar = await NavBarHTML()
+    const messages = await MessageList()
+    const messageForm = await MessageForm()
+    const posts = await PostList()
+
+    const view = getView()
     
+    //Create variable for htmlList = NavBar 
     let htmlList = `${navBar}`
-    htmlList += `
-    <h1>This will be the main page</h1>
-    `
+
+    //check these conditions of the transientState to determine which page to generate 
+        //1. If transientState.view===viewMessages, then invoke function that generates all the HTML for the inbox page and add it to the htmlString
+        if(view === "viewMessages") {
+            htmlList += `<section class="messages
+                    >${messages}<
+                    /section>`
```

Comment:
The browser is reading the </section> HTML as < /section>. If we get rid of the extra space everything else looks good!

Reply:
changes made!